src/config.rs

```diff
@@ -91,6 +92,7 @@ impl Config {
             cached_headers: get_default_headers(&ini),
             cached_log_level: get_default_log_level(&ini),
             cached_vcs_remote: get_default_vcs_remote(&ini),
+            max_retries: obtain_max_retry_count(&ini),
```

Comment:
The other functions with similar functionality are all named `get_…`, I think this one should be as well.